btree/btree.h

```diff
@@ -190,29 +190,6 @@ enum m0_btree_opflag {
 	M0_BOF_UNIQUE = 1 << 0
 };
 
-/**
- * Watermarks for BE space occupied by nodes in lru list.
- */
-enum m0_btree_used_space_watermark{
-	/**
-	 * LRU purging should should not happen below low used space
-	 * watermark
-	 */
-	M0_USW_LOW    = 2 * 1024 * 1024 * 1024ULL,
-
-	/**
-	 * An ongoing LRU purging can be stopped after reaching target used
-	 * space watermark.
-	 */
-	M0_USW_TARGET = 3 * 1024 * 1024 * 1024ULL,
-
-	/**
-	 * LRU purging should be triggered if used space is above high
-	 * used space watermark.
-	 */
-	M0_USW_HIGH   = 4 * 1024 * 1024 * 1024ULL,
-};
-
```

Comment:
We need these enums moved in btree.c to assign as defaults to the variables.

Reply:
Okay.